.github/workflows/tests.yml

```diff
@@ -64,10 +65,17 @@ jobs:
       - name: Generate Xcode projects and compare against existing record
         run: |
           set -euo pipefail
+          bazelisk clean
           bazelisk query 'kind(xcodeproj, tests/ios/xcodeproj/...)' | xargs -n 1 bazelisk run
           bazelisk query 'attr(executable, 1, kind(genrule, tests/ios/xcodeproj/...))' | xargs -n 1 bazelisk run
           git diff --exit-code tests/ios/xcodeproj
+      - name: Ensure hmap files do not exist (or remove if they do)
+        run: |
+          grep "HEADER_SEARCH_PATHS" tests/ios/xcodeproj/*.xcodeproj/project.pbxproj | grep -o  "bazel-out\S*\.hmap" | xargs -I {} bash -c 'F=$0; echo $F; if [ -f $F ]; then rm $F; echo "removed file at $F"; fi' "{}"
```

Comment:
I think this was brought by Sam earlier ... but can you move these steps into the build.sh ? Thanks.

Reply:
Also, I think these tests should all be referring to FRAMEWORK_SEARCH_PATHS, not HEADER_SEARCH_PATHS.

This is because in the lldb-settings.sh, we tell LLDB about the $FRAMEWORK_SEARCH_PATHS.

Reply:
for now it does the same thing of checking against the hash value to see if what xcodeproj wants is what bazel build generates inside `build-warapper.sh`
This test case will fail if changes in this PR is reverted (the transitive rule) so i do see value in this test